packages/test-kit/package.json

```diff
@@ -4,7 +4,9 @@
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "scripts": {
-    "test": "mocha \"test/**/*.unit.ts?(x)\""
+    "test": "yarn test:unit && yarn test:spec",
```

Comment:
why is the separation needed? call the test file `.unit`

Reply:
ok, renamed